UPGRADE_NOTES.md

```diff
@@ -27,8 +27,7 @@ brew upgrade lucky
 ### General updates
 
 - Update: `params.get` now strips white space. If you need the raw value, use `params.get_raw`.
-- Rename: `mount` to `m` in all pages that use components.
-- Update: all mounted components to use new signature `mount(MyComponent.new(x: 1, y: 2))` -> `m(MyComponent, x: 1, y:2)`.
```

Comment:
I think we should keep this all as-is since the signature still needs to change and so that people can upgrade one version at a time if needed

Reply:
@paulcsmith Yeah, I was thinking that removing it would be okay since nothing actually _breaks_ and you just get deprecation warnings. Totally see the value in having each upgrade result in a clean, no-deprecation-warning app though. In this case if they go one by one it'll just mean a rename from `mount` -> `m` -> `mount`, but having done that myself I don't think it's too bad.

Are you good with me keeping the bolded note so that folks know it was later reverted?